keyboards/maxr1998/pulse4k/readme.md

```diff
@@ -0,0 +1,14 @@
+# Pulse 4k
+
+*Image coming soon*
```

Comment:
```suggestion
```
Not sure about what others think but id recommend not leaving this in case its forgotten about which im sure it wont be in the future just stays more clean

Reply:
Fair point, will remove it then, it definitely looks cleaner without.